packages/elements/src/chart/elements/chart.ts

```diff
@@ -138,9 +138,9 @@ export class Chart extends BasicElement {
   }
 
   /**
-   * List of available chart colors
+   * List of line colors
```

Comment:
line colors won't apply to some chart types such as pie & bubble charts.
```suggestion
   * List of available colors for chart styling
```